src/daria/corrections/shape/curvature.py

```diff
@@ -76,19 +76,11 @@ def __init__(
                 self.config = copy.deepcopy(config)
             elif isinstance(config, str):
                 with open(Path(config), "r") as openfile:
-                    tmp_config = json.load(openfile)
-                if "curvature_correction" in tmp_config:
-                    self.config = tmp_config["curvature_correction"]
-                else:
-                    self.config = tmp_config
+                    self.config = json.load(openfile)
             else:
                 assert isinstance(config, Path)
                 with open(config, "r") as openfile:
-                    tmp_config = json.load(openfile)
-                if "curvature_correction" in tmp_config:
-                    self.config = tmp_config["curvature_correction"]
-                else:
-                    self.config = tmp_config
+                    self.config = json.load(openfile)
```

Comment:
This is redundant. If `config` is of type `Path`, then `Path(config)` is still of type `Path`. So the case that`config` is of type `str` or `Path` can be covered by the above case.

Reply:
I removed it thanks

Reply:
Great!